verification/curator-service/api/src/controllers/access.ts

```diff
@@ -0,0 +1,33 @@
+import { curry } from 'ramda';
+
+const accessControl = (
```

Comment:
Why do we need that RBAC config?
I was envisioning to just be explicit about roles in the express route config defined in index.ts.
For example if we want to restrict GET /api/cases to readers we should do:

```
apiRouter.get('/cases', mustBeAuthenticated(), mustHaveRoles(["reader"]), casesController.list);
```

Reply:
Ah, interesting, I hadn't thought of that. Then there's no question of where the RBAC lives, it's in the source. I'll take that approach, thanks.

Reply:
I've made that change, it's much simpler now!